### PR TITLE
bosh cli should present the full details of the vm state

### PR DESCRIPTION
**Pull request by zhang-hua**
[#100344890](https://www.pivotaltracker.com/story/show/100344890)

Signed-off-by: Hua Zhang <zhuadl@cn.ibm.com>

### CODE REVIEW

**Comment by cppforlife** [comment]
I've updated the story with some changes to how CLI should behave.

File: spec/integration/cli_instances_spec.rb [hunk 1]
```diff
@@ -50,12 +48,10 @@
         |   process-1 | running |               |             |
         |   process-2 | running |               |             |
         |   process-3 | failing |               |             |
-        +-------------+---------+---------------+-------------+
```
Comment (cppforlife) [inline spec/integration/cli_instances_spec.rb]: why was the separator removed? we still want to the separator between instances if --ps is specified.

File: bosh_cli/lib/cli/commands/instances.rb [hunk 2]
```diff
@@ -48,6 +48,8 @@ def show_deployment(name, options={})
 
       row_count = 0
       has_disk_cid = instances[0].has_key?('disk_cid')
+      has_uptime = instances[0]['processes'] && instances[0]['processes'][0].has_key?('uptime')
```
Comment (cppforlife) [inline bosh_cli/lib/cli/commands/instances.rb]: what if there are not processes?

File: bosh_cli/lib/cli/commands/instances.rb [hunk 3]
```diff
@@ -48,6 +48,8 @@ def show_deployment(name, options={})
...
+      has_cpu = instances[0]['processes'] && instances[0]['processes'][0].has_key?('cpu')
```
Comment (cppforlife) [inline bosh_cli/lib/cli/commands/instances.rb]: what if there are not processes?

File: spec/integration/cli_instances_spec.rb (hunk 1 above)
Comment (zhang-hua) [inline spec/integration/cli_instances_spec.rb]: @cppforlife , ok, misunderstand it just for --ps. will update it.

File: bosh_cli/lib/cli/commands/instances.rb (hunk 2 above)
Comment (zhang-hua) [inline bosh_cli/lib/cli/commands/instances.rb]: @cppforlife , good point, will fix it.

File: bosh_cli/lib/cli/commands/instances.rb (hunk 3 above)
Comment (zhang-hua) [inline bosh_cli/lib/cli/commands/instances.rb]: @cppforlife , will fix it.

**Pull request merged**